Tests/VauxTests/VauxTests.swift

```diff
@@ -293,6 +293,106 @@ final class VauxTests: XCTestCase {
     }
   }
   
+    
+  func testLinkScript() {
+    func masterPage() -> HTML {
+        html {
+            body {
+                linkScript(src: "script.js")
+            }
+        }
+    }
+    let correctHTML = """
+        <!DOCTYPE html>
+        <html>
+          <body>
+            <script src="script.js">
+            </script>
+          </body>
+        </html>
+        """.replacingOccurrences(of: "\n", with: "")
+    let vaux = Vaux()
+    vaux.outputLocation = .file(name: "testing", path: "/tmp/")
+    do {
+        let rendered = try renderForTesting(with: vaux, html: masterPage())
+        XCTAssertEqual(rendered, correctHTML)
+    } catch let error {
+        XCTFail(error.localizedDescription)
+    }
+  }
+    
+  func testScript() {
+    var pageTitle = "Page title"
+    var pageBody = "Page body"
+    func pageWithJavascript() -> HTML {
+        html {
+            body {
+                paragraph { "" }.id("script")
+                script(code: "document.getElementById('script').innerHTML = 'Hello JavaScript!';")
```

Comment:
I love this example. I think this is going to be super powerful to show off. Friendly reminder about the 2-space tab width 😄